warehouse/models/payments_views/tests/payments_reliability_mislabeled_routes.sql

```diff
@@ -0,0 +1,114 @@
+WITH payments_rides AS (
+    SELECT * FROM {{ ref('payments_rides') }}
+),
+
+route_z_rides AS (
+    SELECT * FROM payments_rides
+    WHERE route_id = 'Route Z'
+),
+
+null_rides AS (
+    SELECT * FROM payments_rides
+    WHERE route_id IS NULL
+),
+
+payments_tests_monthly_date_spine AS (
+    SELECT * FROM {{ ref('payments_tests_monthly_date_spine') }}
+),
+
+count_route_z_rides AS (
+    SELECT
+
+        participant_id,
+        month_start,
+        COUNT(*) AS n_route_z_rides
+
+    FROM payments_tests_monthly_date_spine
+    INNER JOIN route_z_rides
+        USING (participant_id) WHERE transaction_date_pacific >= month_start AND transaction_date_pacific <= month_end
```

Comment:
is there a reason to use `using` and `where` over `ON ... AND ...`? same elsewhere